README.md

```diff
@@ -19,6 +19,47 @@ You can use `stack build` to build `baseDAO` executable.
 
 Run `stack exec -- baseDAO --help` to see available commands.
 
+### Deploying a contract
+
+<!--
+TODO: simplify commands once we use changes in this MR:
+https://gitlab.com/morley-framework/morley/-/merge_requests/652
+-->
+
+You can dump the entire contract code into a `BaseDao.tz` file using the following command:
+
+```sh
+stack exec baseDAO -- print -n baseDAO
+```
+
+For now this can only produce a contract with empty proposal metadata and the simplest possible configuration.
+
+The initial storage of the contract can be produced using
+
+
+```sh
+admin=<administrator address>
+stack exec baseDAO -- storage-BaseDAO --admin $admin > storage
+```
+
+Later the contract can be originated with `tezos-client`:
+
+```sh
+tezos-client originate contract baseDAO transferring 0 from alice running BaseDAO.tz --init "$(<storage)" --burn-cap 17.0
+```
+
+How to install `tezos-client` executable can be found at [`tezos-package` repository](https://github.com/serokell/tezos-packaging) page.
```

Comment:
1. It's called `tezos-packaging`.
2. I think we should refer to official documentation as well: http://tezos.gitlab.io/introduction/howtoget.html. There are other ways that some people may prefer. And probably `tezos-packaging` will be mentioned there soon and we won't need to refer to `tezos-packaging`.